x-pack/plugins/cases/docs/openapi/components/schemas/case_response_properties.yaml

```diff
@@ -43,15 +43,16 @@ properties:
     type: array
     maxItems: 10000
     items:
-      discriminator:
-        propertyName: type
       oneOf:
         - $ref: 'alert_comment_response_properties.yaml'
         - $ref: 'user_comment_response_properties.yaml'
+      discriminator:
+        propertyName: type
```

Comment:
Don't we have other types that might be returned @cnasikas ? Like `ExternalReferenceAttachmentAttributesRt` or `PersistableStateAttachmentAttributesRt`?

The GET case API returns `CaseRt`(`x-pack/plugins/cases/common/types/domain/case/v1.ts`) that has the property `comments: rt.array(AttachmentRt)`, and this `AttachmentRt` can be more than just these `user` or `alert` comments.

(I think this is out of the scope of this PR so I'll approve anyway.)

Reply:
Good question. We decided with Jonathan in the past to not document them to be able to do some breaking changes if needed in these types. Ideally, IMO, these types should not be returned to users at all.